src/lib/support/logging/CHIPLogging.h

```diff
@@ -109,6 +109,7 @@ enum LogModule
     kLogModule_Shell,
     kLogModule_DeviceLayer,
     kLogModule_SetupPayload,
+    kLogModule_AdditionalDataPayload,
```

Comment:
This looks unused.  And in general, I'm not sure we need that fine-grained a module.

Reply:
good point, it was used in earlier versions, now it's not

Reply:
fixed